src/outbackcdx/Capture.java

```diff
@@ -123,6 +131,10 @@ private void decodeValueV2(ByteBuffer bb) {
         robotflags = VarInt.decodeAscii(bb);
     }
 
+    private void decodeValueV3(ByteBuffer bb) {
+        decodeValueV2(bb);
```

Comment:
This seems to be missing the logic to actually encode and decode the new fields?

Reply:
I didn't fully understand quite what this was doing, it worked for what I was testing it for but some guidance on how to get this right would be appreciated, the last version of Java I used was Java 6. Chalk this up to a misunderstanding and obviously need to fix this. I may need some assistance doing this if I may?

Reply:
I have attempted to resolve this to the best of my understanding, based on my specific use case testing it seems to work, but again, further input would be appreciated.